### ISSUE TITLE
Host (or provide a subdomain for) Alexandre's image reporting tool

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-infrastructure#125]
### What

- [ ] Host (or provide a subdomain for) for @alexfauquette 's image reporting tool

```
On my personal server (amathjourney.com/api), I have an API do save flagged images:

PUT https://amathjourney.com/api/off-annotation/flag-image/[code] with body containing {imgid, url}. The imgid must contain a number (the id of the image) and url is the image src
GET https://amathjourney.com/api/off-annotation/flag-image/[code] will return all the data you put for a given code
DELETE https://amathjourney.com/api/off-annotation/flag-image/[code] with body containing {imgid}
Basically, it saves all that in a database whit the primary key code + imgid
```

**Comment by alexgarel** [comment]
I think it's better to have a docker and then host it on our stagging / prod docker VM.

@alexfauquette do you have a repo for this ? Do you need help to make have a docker-compose for it ?

**Comment by alexfauquette** [comment]
No repo, gandi provide an automatic deployement working with a git installed on your VM ^^

Since for now it's only 1 CRUD ressource written in JS, I think it's would be better to rewrite it in Python to avoid introducing a new back-end language

Already did some Django, sounds robotoff is using Falcon, and folksonomy uses FastAPI.

I think I will start from folksonomy structure, but starting a new project require some free time ^^

**Comment by CharlesNepote** [comment]
Is it still needed?

**Comment by alexfauquette** [comment]
I assume this will be integrated in the nutripatrol project @raphael0202 ?

**Comment by teolemon** [comment]
Yes, it will
